packages/react-core/src/components/Tabs/examples/TabsVerticalExpandableUncontrolled.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Tabs, Tab, TabTitleText } from '@patternfly/react-core';
+export const TabsVerticalExpandableUncontrolled: React.FunctionComponent = () => {
+  const [activeTabKey, setActiveTabKey] = React.useState<string | number>(0);
+  // Toggle currently active tab
+  const handleTabClick = (
+    event: React.MouseEvent<any> | React.KeyboardEvent | MouseEvent,
+    tabIndex: string | number
+  ) => {
+    setActiveTabKey(tabIndex);
+  };
```

Comment:
@tlabaj should this state and handleTabClick both be removed to match the other Uncontrolled example?

Reply:
@thatblindgeye @tlabaj Pinging so this issue doesn't get lost!